webapi/src/main/scala/org/knora/webapi/slice/admin/domain/model/KnoraProject.scala

```diff
@@ -127,6 +134,9 @@ object KnoraProject {
 
   object Description {
 
+    implicit val factory: LangString => Either[String, Description] =
```

Comment:
While it is convenient to have them in the domain objects Companion objects I am not sure if they should contain the knowledge how to render themselves from a database object.

Maybe we find a way to move these to the `org.knora.webapi.slice.common.repo` package?

Reply:
done